tests/runner.py

```diff
@@ -1697,8 +1703,13 @@ def reftest(self, expected, manually_trigger=False):
       }
 ''' % (reporting.read(), basename, int(manually_trigger)))
 
-  def compile_btest(self, args):
-    run_process([EMCC] + args + ['--pre-js', path_from_root('tests', 'browser_reporting.js')])
+  def compile_btest(self, args, include_global_reporting=False):
```

Comment:
i think it would be simpler to avoid adding the new parameter here, and just add the flags from the one test that needs it.

Reply:
Ok, sounds good